cfpb/sbl-frontend
# Content updates (Auth landing, Update institution profile)

Closes #353 

## Changes

- Update institution profile
  - Update intro paragraph content
  - Update RSSD label
  - Reorder fields
- Authenticated landing - Review your financial institution profile
  - Update intro paragraph content


## How to test this PR

1. Visit http://localhost:8899/landing
2. Verify content
3. Visit http://localhost:8899/institution/123456789TESTBANK123/update (or any institution you are associated with)
2. Verify content

## Screenshots

### Auth. Landing page
![screencapture-localhost-8899-landing-2024-04-01-16_42_34](https://github.com/cfpb/sbl-frontend/assets/2592907/2cf97082-8351-4835-85bc-8604479b0238)

### Update institution profile
![screencapture-localhost-8899-institution-123456789TESTBANK123-update-2024-04-01-16_42_26](https://github.com/cfpb/sbl-frontend/assets/2592907/bdabdc5b-030e-4655-9807-cced8692d6fe)
